redisdb/datadog_checks/redisdb/redisdb.py

```diff
@@ -457,9 +457,15 @@ def _check_slowlog(self):
         try:
             slowlogs = conn.slowlog_get(max_slow_entries)
         except TypeError as e:
-            # https://github.com/andymccurdy/redis-py/issues/1475
+            # This catch is needed in PY2 because there is a known issue that has only been fixed after redis
+            # dropped python 2 support
+            # issue: https://github.com/andymccurdy/redis-py/issues/1475
+            # fix: https://github.com/andymccurdy/redis-py/pull/1352
+            # TODO: remove once PY2 is no longer supported
             self.log.exception(e)
-            self.log.error('There was an error retrieving slowlog, these metrics will be skipped')
+            self.log.error(
+                'There was an error retrieving slowlog, these metrics will be skipped. This issue is fixed on Agent 7+'
```

Comment:
add link to https://docs.datadoghq.com/agent/versions/upgrade_to_agent_v7/?tab=linux